src/lua/image.c

```diff
@@ -509,6 +509,9 @@ int dt_lua_init_image(lua_State *L)
   luaA_struct_member(L, dt_image_t, filename, const char_filename_length);
   luaA_struct_member(L, dt_image_t, width, const int32_t);
   luaA_struct_member(L, dt_image_t, height, const int32_t);
+  luaA_struct_member(L, dt_image_t, orientation, const int32_t);
+  luaA_struct_member(L, dt_image_t, aspect_ratio, float);
```

Comment:
You only want to read the value.  Exposing it this way allows it to be written.  For instance: You have a landscape photo and you get the aspect_ratio and it's 1.5.  You have a line in your script such as `image.aspect_ratio = 1.`.  The aspect_ratio is now set to square, but the image hasn't changed and still displays as landscape.  Just the sorting would be incorrect.

The better way to do this to add a get_aspect_ratio member function to read and return the value.

Reply:
Tried declaring it as const and that works.  If you try and write to it, lua throws an error

```
     7.5432 LUA aspect ratio is 1.5
     7.5433 LUA ERROR : /home/bill/.config/dttest13695/lua/contrib/clear_GPS.lua:71: field "aspect_ratio" can't be written for type dt_lua_image_t

stack traceback:
	[C]: in ?
	[C]: in metamethod 'newindex'
	/home/bill/.config/dttest13695/lua/contrib/clear_GPS.lua:71: in upvalue 'clear_GPS'
	/home/bill/.config/dttest13695/lua/contrib/clear_GPS.lua:85: in function </home/bill/.config/dttest13695/lua/contrib/clear_GPS.lua:85>
	[C]: in ?


So, to sum up I'd be fine with losing the orientation and declaring aspect_ratio as const.
```

Reply:
I did'nt intend to have it as a read/write properties. I will fix that.